src/main/scala/co/teapot/tempest/algorithm/MonteCarloPPR.scala

```diff
@@ -51,6 +51,7 @@ object MonteCarloPPR {
                   firstStepDirection: EdgeDir,
                   lengthConstraint: LengthConstraint,
                   params: MonteCarloPageRankParams,
+                  alternatingWalk: Boolean,
```

Comment:
Sounds like `alternatingWalk` is no longer part of params. But on line 32, it still says that. Can you fix that?